test/database_test.go

```diff
@@ -204,13 +193,12 @@ func TestDatabase_UserDelete(t *testing.T) {
 	}
 }
 
-// initEnv sets the environment variables to what we have defined in Makefile.
-func initEnv() {
-	e1 := os.Setenv("SKYNET_DB_HOST", "localhost")
-	e2 := os.Setenv("SKYNET_DB_PORT", "37017")
-	e3 := os.Setenv("SKYNET_DB_USER", "admin")
-	e4 := os.Setenv("SKYNET_DB_PASS", "ivolocalpass")
-	if err := errors.Compose(e1, e2, e3, e4); err != nil {
-		panic(err)
+// DBTestCredentials sets the environment variables to what we have defined in Makefile.
+func DBTestCredentials() database.DBCredentials {
+	return database.DBCredentials{
+		User:     "admin",
+		Password: "ivolocalpass",
```

Comment:
probably best to not commit our names but rather use something generic

Reply:
Will change in #8.